### ISSUE TITLE
Mini Batch Gradient Descent

### ISSUE THREAD

**Issue by marshalljacobs12** [marshalljacobs12/Machine-Learning#7]
Implement def miniBatchGradientDescent(model, momentum, _lambda, _alpha, _learning_rate) with and without momentum
